src/clientiface.cpp

```diff
@@ -60,6 +60,23 @@ void RemoteClient::ResendBlockIfOnWire(v3s16 p)
 	}
 }
 
+LuaEntitySAO *getAttachedObject(PlayerSAO *sao, ServerEnvironment *env) {
+	if (!sao->isAttached()) {
```

Comment:
no need for braces, @sfan5 will be angry :p

Reply:
Also: function's brace belongs to a separate new line